tests/test_ex3.py

```diff
@@ -0,0 +1,28 @@
+from pytest_mock import MockerFixture
+import src.ex3
+
+
+def test_get_description():
+    # Assert
+    assert src.ex3.get_description('BarackObama') == src.ex3.BIOS['BarackObama']
+    assert src.ex3.get_description('SandroPertini') == src.ex3.BIOS['SandroPertini']
+    assert src.ex3.get_description('NelsonMandela') == src.ex3.BIOS['NelsonMandela']
+    assert src.ex3.get_description('MahatmaGandi') == src.ex3.BIOS['MahatmaGandi']
```

Comment:
ho sbagliato a scrivere Gand**h**i mi sà

Reply:
comunque, forse potresti fare un ciclo for per testare questa funzione, tipo così https://github.com/UNICT-Quality-Development/qd-starter-pack/pull/7/files